lib/main.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import bedrock from 'bedrock';
+const {config, util: {extend}} = bedrock;
+import fs from 'fs';
+import jsYaml from 'js-yaml';
+import logger from './logger.js';
+import path from 'path';
+
+import './config.js';
+
+const namespace = 'config-yaml';
+const cfg = config[namespace];
+
+bedrock.events.on('bedrock.start', () => {
```

Comment:
don't we have bedrock config stuff we need before `bedrock.started` occurs?

https://github.com/digitalbazaar/bedrock#bedrockevents

such as `bedrock.init` for instance?

Reply:
@aljones15 noting that you said `bedrock.started` and the code is `bedrock.start`.  Does that clarify anything for you?  I do believe `bedrock.start` is the appropriate event.

Reply:
`bedrock.start` is probably early enough so I'll mark this resolved.